learn/Basic/008_function/functionAnonymous.js

```diff
@@ -0,0 +1,7 @@
+// == Contoh == //
```

Comment:
Ditambahin eslint disable `wrap-iife` buat matiin eslint errornya

```suggestion
/* eslint-disable wrap-iife */
// == Contoh == //
```